### PR TITLE
Tl/aim routes

### PR DESCRIPTION
**Pull request by trevorhere**

### CODE REVIEW

File: client/services/AimService.tsx [hunk 1]
```diff
@@ -102,7 +102,14 @@ export const getAim = async (id: string) => {
 
 export const getAims = async () => {
   try {
-    const res = await fetch(`${process.env.NEXT_PUBLIC_API_URL_SSR}/aims`)
+    return [
```
Comment (connorlindsey) [inline client/services/AimService.tsx]: Why are we hardcoding this?

File: client/services/AuthService.tsx [hunk 2]
```diff
@@ -10,6 +22,7 @@ interface User {
 }
 
 const AuthService = ({ children }) => {
+  const { authenticationToken, setAuthenticationToken } = useContext(authContext)
```
Comment (connorlindsey) [inline client/services/AuthService.tsx]: Kinda confused by this. AuthService is already a context provider. I think we just need to useState here (see `useAuth`)

**Review by connorlindsey — CHANGES_REQUESTED** [review]
Few more changes

File: client/services/AuthService.tsx (hunk 2 above)
Comment (trevorhere) [reply client/services/AuthService.tsx]: i deleted this line, not sure why it's showing here.

Comment (trevorhere) [reply client/services/AuthService.tsx]: shows as outdate, nvm

File: client/services/AimService.tsx (hunk 1 above)
Comment (trevorhere) [reply client/services/AimService.tsx]: we still need to figure out how to get a token or id into this function, returned hardcoded aim bc that isn't done yet. I removed the hardcoded return, but the original problem still exists.

**Pull request merged**